### PR TITLE
Register a default token provider to allow admin to rest password.

### LINKED ISSUE #16074: No IUserTwoFactorTokenProvider<TUser> named 'Default' is registered.
### Describe the bug
By default the users' admin controller expose a way for the admin to reset passwords. But by default we do not register any token provider. This causes an exception during the post request

> No IUserTwoFactorTokenProvider<TUser> named 'Default' is registered.

### To Reproduce
1. disable two factor authentication features, and password reset feature.
2. do to the admin, find a user, and try to reset their password.
3. See error

I'll fix this issue soon.

### Screenshots
image

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #16074

### CODE REVIEW

**Review by Piedone — APPROVED** [review]

**Comment by hishamco** [comment]
Do we approve before or after merge  :)

**Comment by Piedone** [comment]
Before :D. But apparently, Mike merged the PR while I was reviewing and seconds before I approved.

**Pull request merged**
